caffe2/utils/math_test.cc

```diff
@@ -426,253 +426,6 @@ TEST(MathTest, FloatToHalfConversion) {
 
 namespace {
 
-class ReduceTensorTest : public testing::Test {
```

Comment:
Where do these tests go?

Reply:
This part is actually covered by reduce_ops_test. So there is no need to keep such cc_tests.